pkg/jobrunaggregator/jobrunaggregatorapi/types_row_jobrun.go

```diff
@@ -0,0 +1,65 @@
+package jobrunaggregatorapi
+
+import "time"
+
+const (
+	LegacyJobRunTableName     = "JobRuns"
+	DisruptionJobRunTableName = "BackendDisruption_JobRuns"
```

Comment:
Please add a comment on both of these table names explaining what goes into each.